cul.js

```diff
@@ -105,10 +105,15 @@ var Cul = function (options) {
     serialPort.on("open", function () {
 
         if (options.init) {
-            that.write(options.initCmd, function (err, res) {
+            that.write(options.initCmd, function (err) {
                 if (err) throw err;
+            });
+            serialPort.drain(function(err){
```

Comment:
yeah you're completely right, it would be enough to use it in the Cul.write method. I tried it step for step,  therefore I started from the top...